Cargo.toml

```diff
@@ -15,7 +15,7 @@ include = ["bindings/rust/*", "grammar.js", "queries/*", "src/*"]
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = "0.22"
+tree-sitter-language = "0.1.0"
```

Comment:
What is `tree-sitter-language`, and why is it replacing `tree-sitter`? The repo link on crates.io just points back to the tree-sitter page.

Reply:
(merging without an answer because I get this same output from `tree-sitter generate`, and also your name is Max Brunsfeld)

Reply:
That crate just defines a language data type; it’s a way of avoiding depending on a particular version of the tree-sitter library. Now, both the tree-sitter crate and the generated language crates depend on that tiny shared crate, so that they can have a type-safe interface with each other without a dependency